### ISSUE TITLE
Blank buttons in keepassxc

### ISSUE THREAD

**Issue by outpaddling** [lumina-desktop/lumina#764]
Not suggesting this is a Lumina bug, but it only happens under Lumina and hoping someone here might have an idea what's causing it.  Under Lumina, all the task bar buttons and buttons on entry screens such as "Toggle password" are all blank.  The problem does not occur under a fluxbox-only install, XFCE, or KDE.
![keepassxc](https://user-images.githubusercontent.com/11299711/119392175-04b43500-bc95-11eb-832c-536c973d3046.png)

**Comment by q5sys** [comment]
Most likely we the icons that are needed for that are not present in the system.  This can happen if they are named uniquely in the program.  We'd need to look into the source for that program to see what icons they are calling and then ensure that they are in the system or there are symlinks for them.

**Comment by outpaddling** [comment]
Well, I don't see the issue in a Fluxbox-only setup.  The only thing desktop-installer was adding under the Fluxbox selection that was not installed with Lumina was fluxbox-tenr-styles-pack.  I tried installing that and restarting X11 for good measure, but the problem persists.
Could it be a conflict?  I reported another icon issue a long time ago that might have a similar cause:
https://github.com/lumina-desktop/lumina/issues/563

**Comment by outpaddling** [comment]
I ran a find /usr/local/share/icons -type f on both systems and verified that the fluxbox-only install has nothing that the Lumina install lacks.  let me know if there is another place I should be looking.

**Comment by q5sys** [comment]
I know there were some outstanding bugs with 1.6.1 that you've reported.  But can you check to see if this presents on that version?  When you initially reported this the FreeBSD port was still on 1.6.0.

**Comment by outpaddling** [comment]
Yep, still blank with 1.6.1.  I'd give this a low priority, though, since it doesn't affect most apps.

My biggest concern by far is the failure to redraw the desktop after a screen resize.  I'd like to be able to recommend Lumina for VirtualBox guests.

**Comment by outpaddling** [comment]
Exploring further, I found that if I change the icon theme under Preferences/Appearance/Theme/Icons, some themes work and others do not:                                          
                                                                                
```
Adwaita                 invisible                                               
Breeze                  ok                                                      
Breeze dark             ok                                                      
La Capitaine            ok                                                      
Lumina icons            same as La Capitaine                                    
Material design light   invisible                                               
Material design dark    invisible                                               
```
